File: tests/fixtures/language_adapter_symbols/sample.php
<?php
// HAND-DERIVED: standard PHP syntax written from language knowledge, not from any adapter regex.

namespace Inventory;

class Item
{
    private string $name;

    public function __construct(string $name, int $quantity)
    {
        $this->name = $name;
    }

    public function getName(): string
    {
        return $this->name;
    }
}

function addQuantity(int $a, int $b): int
{
    return $a + $b;
}

File: tests/fixtures/language_adapter_symbols/sample.cpp
// HAND-DERIVED: standard C++ syntax written from language knowledge, not from any adapter regex.
#include <string>

namespace inventory {

class Item {
public:
    Item(const std::string &name, int quantity) : name_(name), quantity_(quantity) {}

    int quantity() const { return quantity_; }

private:
    std::string name_;
    int quantity_;
};

int add_quantity(int a, int b) {
    return a + b;
}

}  // namespace inventory
